nengo_viz/components/netgraph.py

```diff
@@ -42,12 +153,16 @@ def get_parents(self, uid):
         return parents
 
     def update_client(self, client):
+        self.check_for_reload()
```

Comment:
How often is `update_client` called? Going out to the filesystem to determine the `mtime` is fast in I/O terms, but it's still I/O. If this is called, e.g., 50+ times per second we might only want to check for reload sometimes. If it's not that much, then probably it's fine. Long term, we should probably listen to filesystem events instead of polling (made an issue for that; #182).